tests/base/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+DEAL_II_PICKUP_TESTS()
```

Comment:
I would test explicitly the `partpxa` library here similarly as in https://github.com/hyperdeal/hyperdeal/blob/9bfd4b2189fe27896a782c641a30bda219437ff5/tests/base/CMakeLists.txt#L3